components/automate-ui/src/app/page-components/run-history/run-history.component.ts

```diff
@@ -110,7 +110,7 @@ export class RunHistoryComponent implements OnInit, OnDestroy {
   }
 
   getDuration(start_time, end_time) {
-    return moment.duration(moment.utc(end_time).diff(moment.utc(start_time))).humanize();
+    return moment.duration(moment(end_time).diff(moment(start_time))).humanize();
```

Comment:
this is interesting.
this seems to be calculating the duration of a run. but i would assume, when looking at that panel, that this was trying to tell me how long ago it happened in human terms, not the duration of run..

Reply:
When i print it out to the console - pre AND post change, it prints the correct numbers as well.

Reply:
what makes this extra interesting to me is that...it's always been like this it seems...am i confusing myself or something?

Reply:
nope - it has always been like this ( without `.utc`) - this is essentially just a revert.  Its been like this since the at least the initial open sourcing.

Reply:
hmm. ya it def doesn't make sense to show the duration of a run and then put "ago" at the end...i'm asking ux! :)

Reply:
@SEAjamieD @vjeffrey we would like to keep it as duration for now
but can you please add a `Duration: ` label before the text of the duration (e.g., Duration: 15 seconds)?

@jonong1972 is going to follow up later with another issue exploring having all three pieces of data (duration, relative time, and absolute time)

Reply:
@susanev - I should also remove `ago` text, correct?

Reply:
@SEAjamieD Yes please. thank you!

Reply:
Yea I'd imagine `{{ history.endTime | timeFromNow }}` would be used if we wanted the "ago" text.